content/blog/2021-04-15-april-21-dvc-heartbeat.md

```diff
@@ -0,0 +1,182 @@
+﻿---
+title: April '21 Heartbeat
+date: 2021-04-15
+description: |
+  Monthly updates are here! Loads from the Community, use cases, videos, 5 new hires and more!
+descriptionLong: |
+  Monthly updates are here! Read all about what our community is doing
+  with DVC, our CEO's interviews and talks, new team members and more!
+picture: 2021-04-15/apr21cover.png
+author: jeny_defigueiredo
+commentsUrl: https://discuss.dvc.org/t/april-21-heartbeat/718?u=jendefig
+tags:
+  - Heartbeat
+  - DVC
+  - MLOps
+---
+
+## From the Community
+
+We're starting with the community this month because it has been overflowing
+with great content from our users. It's like we're on a sugar high!
+
+![Sugar High](https://media.giphy.com/media/oiGCnybFPh6Q8/giphy.gif)
+
+### Goku Mohandas Tutorial
+
+First up, [Goku Mahandas](https://twitter.com/GokuMohandas) of
+[Made With ML](https://madewithml.com/) has created
+[this tutorial](https://madewithml.com/courses/mlops/versioning/) using DVC.  
+It's RT'ing around the MLOps Twitter space like hotcakes! 🥞
```

Comment:
Was that new line intentional?
```suggestion
[this tutorial](https://madewithml.com/courses/mlops/versioning/) using DVC.
It's RT'ing around the MLOps Twitter space like hotcakes! 🥞
```

Reply:
p.s. breaks the p:
![image](https://user-images.githubusercontent.com/1477535/114788876-dbc08c00-9d47-11eb-8a2c-ab6667d5e97b.png)